artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -264,6 +264,8 @@
 
    private static final String AMQP_USE_CORE_SUBSCRIPTION_NAMING = "amqp-use-core-subscription-naming";
 
+   private static final String AMQP_USE_MODIFIED_FOR_TRANSIENT_DELIVERY_ERRORS = "amqp-use-modified-for-transient-delivery-errors";
```

Comment:
Same thing here, please use ProtonProtocolManager property for this. you can simply set a boolean property on the acceptors.

Reply:
@clebertsuconic you mean like amqpLowCredits?

`         <acceptor name="amqp">tcp://0.0.0.0:5672?tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;protocols=AMQP;useEpoll=true;amqpCredits=1000;amqpLowCredits=300</acceptor>`

if so would.. be acceptable?
`amqpUseModifiedForTransientDeliveryErrors=<boolean>`

Reply:
@k-wall yes.. but can't you find a shorter property name? :)

Reply:
@clebertsuconic configuration mechanism changed as suggested.  

on the name, I can't think how to shorten significantly without losing important information.  Best alternative I can up with was `amqpSendModifiedForTransientErrors` which save a few characters.  FWIW I don't think too many users will be configuring this option.  I'm happy to rename as you see fit.  Let me know.